FhirToDataLake/azurepipelines/ci.yml

```diff
@@ -25,10 +25,11 @@ steps:
 - task: DotNetCoreCLI@2
   displayName: 'dotnet restore solution'
   inputs:
-    command: restore
-    projects: FhirToDataLake/*.sln
-    feedsToUse: 'config'
-    nugetConfigPath: 'FhirToDataLake/NuGet.Config'
+    workingDirectory: FhirToDataLake
+    command: custom
+    custom: restore
+    arguments: >
+      Microsoft.Health.AnalyticsConnector.Synapse.sln
```

Comment:
DevOps native restore task have bug for **packageSourceMapping** config, refer this [discussion](https://github.com/NuGet/Home/issues/11406)